docs/services/camera_sim.rst

```diff
@@ -1,14 +1,59 @@
 Camera Simulator
 ================
+This service operates a simulated camera. This service is meant to mimic, but does not actually
+control, a hardware camera.
+
+When applicable, all services have a corresponding simulated service to be able to test control
+software before commanding actual hardware devices.
```

Comment:
I think this should go into a different part of the docs, where the services are describe, so I think you can delete it here. Instead of it, you could say that this simulated camera service can be used as the simulated service for any of the hardware camera services, since they're all written consistently.

Reply:
good call, updated.